src/features/spa/aggregate/serializer.js

```diff
@@ -11,177 +11,180 @@ import { getInfo } from '../../../common/config/config'
 export class Serializer extends SharedContext {
   constructor(parent) {
     super(parent)
+    this.firstTimestamp = undefined;
   }
 
   serializeMultiple(interactions, offset, navTiming) {
     const info = getInfo(this.sharedContext.agentIdentifier)
     var addString = getAddStringContext(this.sharedContext.agentIdentifier)
     var serialized = 'bel.7'
     interactions.forEach((interaction) => {
-      serialized += ';' + serializeInteraction(interaction.root, offset, navTiming, interaction.routeChange, addString, info)
+      serialized += ';' + this.serializeInteraction(interaction.root, offset, navTiming, interaction.routeChange, addString, info)
     })
+    this.firstTimestamp = undefined
```

Comment:
Could you just add a jsdoc or some comments for context on this variable? (Relating to this issue)